jib-core/src/test/java/com/google/cloud/tools/jib/http/WithServerFailoverHttpClientTest.java

```diff
@@ -70,7 +70,7 @@ public void testGet()
   public void testSecureConnectionOnInsecureHttpsServer()
       throws IOException, InterruptedException, GeneralSecurityException, URISyntaxException {
     FailoverHttpClient secureHttpClient =
-        new FailoverHttpClient(false /*secure*/, false, logger, false);
+        new FailoverHttpClient(false /*secure*/, false, logger, true);
```

Comment:
This one particularly needs turning off retrying. (The test still works without turning off, but it takes a lot of time before retrying finally gives up.)